### PR TITLE
Add option to use openAIGym without pygame rendering

### PR DESCRIPTION
**Pull request by jackdavidweber**
I got the idea that we could really improve performance if we got rid of pygame. So I added an argument to the init of the snake env. Turns out, the efficiency gains are significant!

We see that training and testing occurs about 4.6 times faster when pygame is not being used:
Screen Shot 2021-11-01 at 12 35 44 AM

From the couple times that I timed it, the efficiency gains ranged from 3-9x

## Why is this important?
- Faster training means we can try more things in less time
- Will encounter fewer bugs when trying to run this on the server

### CODE REVIEW

**Comment by jackdavidweber** [comment]
https://tree.taiga.io/project/jackdavidweber-snnake/us/39?milestone=302421

File: gym_snake/envs/snakeGameGym.py [hunk 1]
```diff
@@ -16,19 +18,37 @@ class SnakeGameGym(SnakeGame):
...
 		"""
+		self.use_pygame = use_pygame
 		self.move_map = {
 			0: "left",
 			1: "up",
 			2: "right",
 			3: "down",
 		}
+		
+		self.width = 500
```
Comment (davidd-55) [inline gym_snake/envs/snakeGameGym.py:33]: Is there a particular reason for adding all of these instance variables to the SnakeGameGym class proper? I'm fine with it, but it seems to be adding duplicate code that isn't super necessary.

**Comment by davidd-55** [comment]
This is super sick! Sweet that there's so much speed improvement on the server when running now. 

I just left one comment with a question about code organization/duplication. Other than that, LGTM!

**Review by davidd-55 — APPROVED** [review]

**Review by dv3-000 — APPROVED** [review]
Looks good! Love a good speed-up

**Pull request merged**